Dockerfile.heroku-18-libssl1.0

```diff
@@ -4,9 +4,10 @@ WORKDIR /app
 ENV WORKSPACE_DIR="/app/builds" \
     S3_BUCKET="lang-python" \
     S3_PREFIX="heroku-18/" \
-    DEBIAN_FRONTEND=noninteractive
+    DEBIAN_FRONTEND=noninteractive \
```

Comment:
This is good, I wonder if we should also add a note at the top in both files to remind anyways editing either one of them, to make the same change on the other one ?

Reply:
Good call! I will add the note.

Reply:
Added.